diagnostics-extension/src/services/telemetry.ts

```diff
@@ -76,6 +76,9 @@ export class TelemetryServiceImpl extends TelemetryService {
   async getOemData(): Promise<OemData> {
     return dpsl.telemetry.getOemData();
   }
+  async getCpuInfo(): Promise<CpuInfo> {
+    return dpsl.telemetry.getCpuInfo();
```

Comment:
Should we bump the dps version in `package.json`? I am not 100% sure whether the current version supports CPU info.